<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaOverlay</class>
 <widget class="QWidget" name="MediaOverlay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>355</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#MediaOverlay {
	background-color: transparent;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>258</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frameMediaControls">
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#frameMediaControls {
	background-color: rgba(180, 180, 180, 140);
	border-radius: 10px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QToolButton" name="toolButtonPlayPause">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton {
	background: transparent;
	border: 0px solid transparent;
	image: url(:/Resource/Media/PlayMedia.png);
}

QToolButton:hover {
	image: url(:/Resource/Media/PlayMediaH.png);
}

QToolButton:pressed {
	image: url(:/Resource/Media/PlayMediaP.png);
}

QToolButton:disabled {
	image: url(:/Resource/Media/PlayMediaD.png);
}

QToolButton:checked {
	image: url(:/Resource/Media/PauseMedia.png);
}

QToolButton:checked:hover {
	image: url(:/Resource/Media/PauseMediaH.png);
}

QToolButton:checked:pressed {
	image: url(:/Resource/Media/PauseMediaP.png);
}

QToolButton:checked:disabled {
	image: url(:/Resource/Media/PauseMediaD.png);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonPrevious">
          <property name="styleSheet">
           <string notr="true">QToolButton {
	background: transparent;
	border: 0px solid transparent;
	image: url(:/Resource/Media/PreviousMediaTrack.png);
}

QToolButton:hover {
	image: url(:/Resource/Media/PreviousMediaTrackH.png);
}

QToolButton:pressed {
	image: url(:/Resource/Media/PreviousMediaTrackP.png);
}

QToolButton:disabled {
	image: url(:/Resource/Media/PreviousMediaTrackD.png);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonStop">
          <property name="styleSheet">
           <string notr="true">QToolButton {
	background: transparent;
	border: 0px solid transparent;
	image: url(:/Resource/Media/StopMedia.png);
}

QToolButton:hover {
	image: url(:/Resource/Media/StopMediaH.png);
}

QToolButton:pressed {
	image: url(:/Resource/Media/StopMediaP.png);
}

QToolButton:disabled {
	image: url(:/Resource/Media/StopMediaD.png);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonNext">
          <property name="styleSheet">
           <string notr="true">QToolButton {
	background: transparent;
	border: 0px solid transparent;
	image: url(:/Resource/Media/NextMediaTrack.png);
}

QToolButton:hover {
	image: url(:/Resource/Media/NextMediaTrackH.png);
}

QToolButton:pressed {
	image: url(:/Resource/Media/NextMediaTrackP.png);
}

QToolButton:disabled {
	image: url(:/Resource/Media/NextMediaTrackD.png);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonOpen">
          <property name="styleSheet">
           <string notr="true">QToolButton {
	background: transparent;
	border: 0px solid transparent;
	image: url(:/Resource/Media/OpenMedia.png);
}

QToolButton:hover {
	image: url(:/Resource/Media/OpenMediaH.png);
}

QToolButton:pressed {
	image: url(:/Resource/Media/OpenMediaP.png);
}

QToolButton:disabled {
	image: url(:/Resource/Media/OpenMediaD.png);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonReplay">
          <property name="styleSheet">
           <string notr="true">QToolButton {
	background: transparent;
	border: 0px solid transparent;
	image: url(:/Resource/Media/Repeat.png);
}

QToolButton:hover {
	image: url(:/Resource/Media/RepeatH.png);
}

QToolButton:pressed {
	image: url(:/Resource/Media/RepeatP.png);
}

QToolButton:disabled {
	image: url(:/Resource/Media/RepeatD.png);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonShuffle">
          <property name="styleSheet">
           <string notr="true">QToolButton {
	background: transparent;
	border: 0px solid transparent;
	image: url(:/Resource/Media/Shuffle.png);
}

QToolButton:hover {
	image: url(:/Resource/Media/ShuffleH.png);
}

QToolButton:pressed {
	image: url(:/Resource/Media/ShuffleP.png);
}

QToolButton:disabled {
	image: url(:/Resource/Media/ShuffleD.png);
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonFullScreen">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../Resource.qrc">
            <normaloff>:/Resource/Media/FullScreen.png</normaloff>:/Resource/Media/FullScreen.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonSettings">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../Resource.qrc">
            <normaloff>:/Resource/Generic/Settings.png</normaloff>:/Resource/Generic/Settings.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonMute">
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton {
	background: transparent;
	border: 0px solid transparent;
	image: url(:/Resource/Media/Mute.png);
}

QToolButton:hover {
	image: url(:/Resource/Media/MuteH.png);
}

QToolButton:pressed {
	image: url(:/Resource/Media/MuteP.png);
}

QToolButton:checked {
	image: url(:/Resource/Media/Unmute.png);
}

QToolButton:checked:hover {
	image: url(:/Resource/Media/UnmuteH.png);
}

QToolButton:checked:pressed {
	image: url(:/Resource/Media/UnmuteP.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>26</width>
            <height>26</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSliderVolume">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>15</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QSlider" name="horizontalSliderSeek">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelTime">
        <property name="text">
         <string>0:00 / 0:00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections/>
</ui>
